ggml/src/ggml-quants.c

```diff
@@ -4,6 +4,7 @@
 #include "ggml-quants.h"
 #include "ggml-impl.h"
 #include "ggml-cpu/ggml-cpu-impl.h"
+#include "ggml-cpu/simd-mappings.h"
```

Comment:
The sources from `ggml-base` should not depend on the CPU backend. Instead, what I would suggest is keeping `GGML_FP16_TO_FP32` and similar in `ggml-base` (with just the C implementation), and moving optimized versions to the CPU backend, renamed to something like `GGML_CPU_FP16_TO_FP32`.